scheduler/src/cook/mesos/task.clj

```diff
@@ -63,7 +63,7 @@
            (cook-executor-candidate? job-ent))))
 
 ; TODO: This is mesos specific.
-(defn build-executor-environment
```

Comment:
This is probably not actually mesos specific, the kubernetes progress reporter will probably use the same environment variables for backwards compatibility.

Reply:
Ok. Name reverted.